test/cuda/test_execution_engine.cc

```diff
@@ -255,7 +251,7 @@ def convolutionStrided(float(N,C,H,W) I, float(O,C,KH,KW) W1, float(O) B)
   tcStr = tc::replaceString(tcStr, "<sh>", std::to_string(SH));
   tcStr = tc::replaceString(tcStr, "<sw>", std::to_string(SW));
 
-  Check(
```

Comment:
outputs is used as argument of the call to `Check` here, but is not fully defined until `Check` returns.  Maybe you want to _drop_ the `outputs` argument and only use the returned vector?